<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>677</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>311</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QComboBox" name="comboBox_b_select">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>B2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Magnet</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Designed Energy</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_designed_energy">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>92.99</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MeV</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Designed Dispersion</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_profile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Prof B2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_designed_dispersion">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.9877</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Acc.Modules Setup</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>41</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>A2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>A3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>A4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>A5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>A6</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>A7</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>241</width>
       <height>370</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Amp.(MV)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Phase(degree)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLineEdit" name="lineEdit_amp_a2">
         <property name="text">
          <string>40</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_pha_a2">
         <property name="text">
          <string>66.4</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QScrollBar" name="horizontalScrollBar_amp_a2">
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="value">
          <number>400</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QScrollBar" name="horizontalScrollBar_pha_a2">
         <property name="maximum">
          <number>1800</number>
         </property>
         <property name="value">
          <number>900</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit_amp_a3">
        <property name="text">
         <string>40</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_pha_a3">
        <property name="text">
         <string>66.4</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QScrollBar" name="horizontalScrollBar_amp_a3">
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>400</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QScrollBar" name="horizontalScrollBar_pha_a3">
        <property name="maximum">
         <number>1800</number>
        </property>
        <property name="value">
         <number>900</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="lineEdit_amp_a4">
        <property name="text">
         <string>46.5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_pha_a4">
        <property name="text">
         <string>112</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QScrollBar" name="horizontalScrollBar_amp_a4">
        <property name="maximum">
         <number>465</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>465</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QScrollBar" name="horizontalScrollBar_pha_a4">
        <property name="maximum">
         <number>1800</number>
        </property>
        <property name="value">
         <number>900</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="lineEdit_amp_a5">
        <property name="text">
         <string>46.5</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_pha_a5">
        <property name="text">
         <string>112</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QScrollBar" name="horizontalScrollBar_amp_a5">
        <property name="maximum">
         <number>465</number>
        </property>
        <property name="value">
         <number>465</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QScrollBar" name="horizontalScrollBar_pha_a5">
        <property name="maximum">
         <number>1800</number>
        </property>
        <property name="value">
         <number>900</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="lineEdit_amp_a6">
        <property name="text">
         <string>46.5</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit_pha_a6">
        <property name="text">
         <string>112</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QScrollBar" name="horizontalScrollBar_amp_a6">
        <property name="maximum">
         <number>465</number>
        </property>
        <property name="value">
         <number>465</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QScrollBar" name="horizontalScrollBar_pha_a6">
        <property name="maximum">
         <number>1800</number>
        </property>
        <property name="value">
         <number>900</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLineEdit" name="lineEdit_amp_a7">
        <property name="text">
         <string>46.5</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="lineEdit_pha_a7">
        <property name="text">
         <string>112</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QScrollBar" name="horizontalScrollBar_amp_a7">
        <property name="maximum">
         <number>465</number>
        </property>
        <property name="value">
         <number>465</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QScrollBar" name="horizontalScrollBar_pha_a7">
        <property name="maximum">
         <number>1800</number>
        </property>
        <property name="value">
         <number>900</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_set">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>530</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_default">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>530</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>comboBox_b_select</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>lineEdit_designed_energy</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>lineEdit_profile</zorder>
    <zorder>lineEdit_designed_dispersion</zorder>
    <zorder>label_6</zorder>
    <zorder>pushButton_set</zorder>
    <zorder>pushButton_default</zorder>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>130</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pause">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>200</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>450</y>
      <width>160</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Energy (MeV)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_energy"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Energy Spread (%)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_energy_spread">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vbox_mpl1"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vbox_mpl2"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
